css/index.css

```diff
@@ -1,3 +1,345 @@
+q {
+    quotes: none;
+}
+
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+    content: '';
+    content: none;
+}
+
+table {
+    border-collapse: collapse;
+    border-spacing: 0;
+}
+
+/* Set every element's box-sizing to border-box */
+* {
+    box-sizing: border-box;
+}
+
+html,
+body {
+    height: 100%;
+    font-family: 'Titillium Web', sans-serif;
+}
+
+h1,
+h2,
+h3,
+h4,
+h5 {
+    font-family: 'Bangers', cursive;
+    letter-spacing: 1px;
+    margin-bottom: 15px;
+}
+
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+    max-width: 900px;
+    margin: 0 auto;
+}
+
+/*css for nav bar */
+nav {
+    display: flex;
+    padding-top: 30px;
+    padding-bottom: 20px;
+    width: 100%;
+    margin: 0 auto;
+    width: 100%;
+    align-items: center;
+}
+
+.menu {
+    width: 65%;
+    display: flex;
+    justify-content: space-between;
+}
+
+.menu a {
+    text-decoration: none;
+    color: #cfcfd4;
+    transition: ease-in-out 300ms;
+}
+
+.menu a:hover {
+    color: grey;
+}
+
+.menu-logo {
+    width: 30%;
+    display: flex;
+    justify-content: flex-end;
+}
+
+/* style for header */
+.intro-home {
+    display: flex;
+    width: 100%;
+    padding-top: 50px;
+    padding-bottom: 50px;
+}
+
+.intro {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    font-size: 80px;
+}
+
+.intro h1 {
+    text-align: center;
+}
+
+.intro-img {
+    width: 50%;
+    display: flex;
+    justify-content: center;
+}
+
+.logo {
+    cursor: pointer;
+}
+
+button {
+    height: 35px;
+    width: 150px;
+    border: 1px solid black;
+    font-size: 16px;
+    background: white;
+}
+
+/* style for section */
+.box {
+    display: flex;
+    width: 100%;
+    justify-content: space-between;
+}
+
+.box-one {
+    padding-top: 30px;
+    width: 48%;
+}
+
+.middle-img {
+    display: flex;
+    justify-content: center;
+    padding-top: 15px;
+    width: 100%;
+}
+
+.box-two {
+    width: 32%;
+    padding-top: 33px;
+    padding-bottom: 30px;
+}
+
+.middle-content {
+    border-top: 2px solid black;
+    border-bottom: 2px solid black;
+}
+
+/* style for address */
+.address {
+    margin: 0 auto;
+    width: 97%;
+    padding-top: 20px;
+}
+
+address a {
+    padding-top: 20px;
+    text-decoration: none;
+    color: black;
+}
+
+address p:first-child {
+    padding-bottom: 20px;
+}
+
+address p:last-child {
+    padding-top: 20px;
+}
+
+footer {
+    padding-top: 30px;
+    display: flex;
+    justify-content: center;
+    padding-bottom: 20px;
+}
+
+@media(max-width: 800px) {
+
+    nav {
+        margin: 0 auto;
+        display: flex;
+        flex-direction: column-reverse;
+        width: 90%;
+    }
+    .middle-content {
+        margin-left: 15px;
+        margin-right: 15px;
+        padding: 10px;
+    }
+    .menu {
+        width: 100%;
+        padding-top: 20px;
+    }
+.menu-logo {
+    width: 100%;
```

Comment:
Be careful with indentation. You could use prettier extension on VSCode to be sure everything is formatted correctly. 
https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode